data/blog/tclip-updates-092023.mdx

```diff
@@ -0,0 +1,129 @@
+---
+title: tclip updates for September 2023
+date: 2023-09-19
+authors: ['xe']
+tags:
+  - tclip
+  - tsnet
+summary: 'tclip now has a new theme, org-mode rendering, and runs without CGo.'
+images: ['/images/tclip-updates-092023/social.png']
+---
+
+Hey everyone! We've made a bunch of updates to tclip, and we're excited to share them with you. We hope this will help you get the most out of tclip and give you inspiration to make your own addon tools to do whatever it is you want.
+
+[tclip](https://github.com/tailscale-dev/tclip) is a little addon service that gives everyone on your tailnet a place to share small snippets of text, code, log messages, or anything that can fit into a text box. It's like [GitHub Gists](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) but private on your tailnet.
+
+You can read more about tclip and how to install it on GitHub at [tailscale-dev/tclip](https://github.com/tailscale-dev/tclip). I also wrote more about tclip in [the previous tclip article](/blog/tclip).
+
+This release has a few features and changes that were contributed by the community. Thank you all for helping us!
+
+With huge thanks to:
```

Comment:
I think I would shuffle these paragraphs around a little bit. Here's a classic Have You Considered, feel free to take or leave:

We've been hacking on [tclip](https://github.com/tailscale-dev/tclip), the add-on service that gives everyone in your tailnet a place to share small snippets of text and code since we [launched it on this blog last month](/blog/tclip).

As it turns out, people seem to like the idea of a [GitHub Gists](https://docs.github.com/en/get-started/writing-on-github/editing-and-sharing-content-with-gists/creating-gists) that's private on your tailnet, and we've gotten some great community contributions. We're excited to share these updates, and we owe a special round of thanks to: